modules/core/core-task/src/main/java/com/enonic/xp/impl/task/ClusteredTaskManagerImpl.java

```diff
@@ -153,8 +156,9 @@ public void submitTask( final DescribedTask task )
         public boolean select( final Member member )
         {
             return Boolean.TRUE.equals( member.getBooleanAttribute( MemberAttributesApplier.TASKS_ENABLED_ATTRIBUTE_KEY ) ) &&
-                Boolean.TRUE.equals(
-                    member.getBooleanAttribute( MemberAttributesApplier.TASKS_ENABLED_ATTRIBUTE_PREFIX + task.getApplicationKey() ) );
+                ( !SYSTEM_APPLICATION_KEY.equals( task.getApplicationKey() ) ||
+                    member.getBooleanAttribute( MemberAttributesApplier.SYSTEM_TASKS_ENABLED_ATTRIBUTE_KEY ) ) && Boolean.TRUE.equals(
```

Comment:
```suggestion
                    Boolean.TRUE.equals( member.getBooleanAttribute( MemberAttributesApplier.SYSTEM_TASKS_ENABLED_ATTRIBUTE_KEY ) ) ) && Boolean.TRUE.equals(
```

To avoid NPE